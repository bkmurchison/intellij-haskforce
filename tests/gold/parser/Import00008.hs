module Import00008 where

import Control.Natural (type (~>))
