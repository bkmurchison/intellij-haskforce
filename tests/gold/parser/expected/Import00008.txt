Haskell File
  MODULEDECL
    PsiElement(HaskellTokenType.module)('module')
    QCONID
      CONID
        PsiElement(HaskellTokenType.conidRegexp)('Import00008')
    PsiElement(HaskellTokenType.where)('where')
  BODY
    IMPDECL
      PsiElement(HaskellTokenType.import)('import')
      QCONID
        CONID
          PsiElement(HaskellTokenType.conidRegexp)('Control')
        PsiElement(HaskellTokenType..)('.')
        CONID
          PsiElement(HaskellTokenType.conidRegexp)('Natural')
      PsiElement(HaskellTokenType.()('(')
      IMPORTT
        PsiElement(HaskellTokenType.type)('type')
        PsiElement(HaskellTokenType.()('(')
        VARSYM
          PsiElement(HaskellTokenType.Varsym)('~>')
        PsiElement(HaskellTokenType.))(')')
      PsiElement(HaskellTokenType.))(')')